packages/coinstac-ui/app/render/state/root-reducer.js

```diff
@@ -12,8 +11,7 @@ import { reducer as form } from 'redux-form';
 
 const rootReducer = combineReducers({
   auth,
-  computation,
-  computations,
+  computationsState,
```

Comment:
I like the brevity of `computations`, but `computationsState` makes sense here. Wonder if there’s a way to easily refactor computation use into a single collection instead of the collection hanging from `computations` and the “selected” computation at `computation`. Our implementation doesn’t clearly separate entity state from UI state, which is unfortunate.